src/utilities/logging/index.js

```diff
@@ -0,0 +1,33 @@
+import RNFS from 'react-native-fs';
+import moment from 'moment';
+import { LoggerService } from './Service';
+import { logDir, logFileName, logFileDate, logFileSeparator } from './Transport';
+
+const logFileFilter = file => file?.includes(`${logFileSeparator}${logFileName}`);
+
+const getExceedsThresholdFilter = (threshold = [5, 'days']) => fileName => {
+  const thresholdDate = moment().subtract(...threshold);
+  const [maybeDate] = fileName.split(logFileSeparator);
+  const asMoment = moment(maybeDate, logFileDate);
+  return asMoment.isBefore(thresholdDate);
+};
+
+export const tidyLogFiles = async () => {
+  try {
+    const files = await RNFS.readdir(logDir);
+
+    return Promise.all(
+      // All files with the name {date}__log.txt where the date is some date in the format
+      // DD-MM-YY and is at least 5 days ago.
+      files
+        .filter(logFileFilter)
+        .filter(getExceedsThresholdFilter())
+        .map(fileName => RNFS.unlink(`${logDir}/${fileName}`))
+    );
+  } catch (error) {
+    return null;
+    // Going to just ignore errors for now..?
```

Comment:
am thinking that if you don't have permission / ability to delete the logs then the app probably doesn't permission to create logs.. so there couldn't be too much space taken up with log files. 

That would be my only concern - that the tidy is failing but the writing is ok, and we use up too much space on the device with logging. I think that the best option is to swallow the error if the log tidy fails.. rather than alert user or crash app. Perhaps notify bugsnag though? just in case?